src/main/java/io/druid/data/input/MapBasedRow.java

```diff
@@ -159,6 +161,52 @@ public long getLongMetric(String metric)
   }
 
   @Override
+  public int getIntMetric(String metric)
+  {
+    Object metricValue = event.get(metric);
+
+    if (metricValue == null) {
+      return 0;
+    }
+
+    if (metricValue instanceof Number) {
+      return ((Number) metricValue).intValue();
+    } else if (metricValue instanceof String) {
+      try {
+        return Integer.valueOf(((String) metricValue).replace(",", ""));
```

Comment:
The existing code is only part of the parsers at ingestion time. if this was existing code, then I'd be inclined to leave as is, but given that we are introducing new code that touches data that has already been ingested, we should try to do the right thing and not make any assumptions regarding data formats.